exchanges/ftx/ftx_test.go

```diff
@@ -91,11 +91,17 @@ func TestGetOrderbook(t *testing.T) {
 
 func TestGetTrades(t *testing.T) {
 	t.Parallel()
-	_, err := f.GetTrades(spotPair, 0, 0, 200)
+	// test empty market
+	_, err := f.GetTrades("", 0, 0, 200)
+	if err == nil {
+		t.Error(err)
+	}
+	// no limit or time vals
+	_, err = f.GetTrades(spotPair, 0, 0, 0)
 	if err != nil {
 		t.Error(err)
 	}
-	_, err = f.GetTrades(spotPair, 0, 0, 5)
+	_, err = f.GetTrades(spotPair, 1559881511, 1559901511, 5)
```

Comment:
Just a note: This returns no data as this is before FTT traded on FTX so it doesn't really test the limit.

Reply:
Good catch! Updated these on the public endpoint tests and made them configurable for the auth ones